readme-developer.md

```diff
@@ -1,158 +1,121 @@
-   - libpng-devel
-   - libjpeg-turbo-devel
-   - mesa-libGL-devel
-   - glew-devel
-   - openal-soft-devel
-   - libmad-devel
-   - libuuid-devel
-
-Then, from the project root folder, simply type:
+<details>
+<summary>DEB-based distros</summary>
 
 ```
-    $ scons
-    $ ./endless-sky
+g++ cmake ninja-build pkg-info libgl1-mesa-dev libxmu-dev libxi-dev libglu1-mesa-dev
 ```
 
-The program will run, using the "data" and "images" folders that are found in the project root. For more Linux help, and to view other command-line flags, consult the `man` page (endless-sky.6), by running `man endless-sky` in the project directory.
-The program also accepts a `--help` command line flag.
+If you're on an older version of Ubunut, you'll need an up-to-date version of cmake which you can get from the [offical apt](https://apt.kitware.com/).
+
+</details>
 
-To compile and also run unit tests, the "test" target can be used:
+<details>
+<summary>RPM-based distros</summary>
 
 ```
-  $ scons test
+gcc-c++ cmake ninja-build mesa-libGL-devel
 ```
 
-The produced test binary will be invoked with a reasonable set of arguments. To run any test benchmarks, you will need to invoke the test binary directly. Refer to the build pipeline definitions for the current required arguments.
-More information on unit tests can be found in the project's "tests" directory, or by running the binary with the `--tests` flag.
-
-
+</details>
 
 ## Windows
 
-Two methods of building the game on Windows are supported. The official build is done on the command-line with the Scons program, but the game may also be built using the Code::Blocks IDE. Both require a MinGW toolchain and the precompiled libraries linked below.
-
-MinGW provides the tools used to compile the source code into the game's executable. For most cases, you should download the [MinGW-W64 Online Installer](https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/installer/mingw-w64-install.exe). When installing, use the following settings:
-
-- Version: `8.1.0` (or greater; this document assumes 8.1.0)
-- Architecture: `x86_64`
-- Threads: `posix`
-- Exception: `seh`
-
-Endless Sky requires precompiled libraries to compile and play: [Download link](https://endless-sky.github.io/win64-dev.zip)
-
-The zip can be extracted anywhere on your filesystem; to minimize additional configuration, you can move the `dev64` folder to `C:\`.
-
+Currently, only the MinGW compiler toolchain is supported on Windows. You can download the [MinGW Winlibs](https://winlibs.com/#download-release) build, which also includes various tools you'll need to build the game as well.
 
-### Building with Scons
+You'll need the MSVCRT runtime version, 64-bit. The latest version is currently gcc 12 and can be downloaded [here](https://github.com/brechtsanders/winlibs_mingw/releases/download/12.1.0-14.0.4-10.0.0-msvcrt-r2/winlibs-x86_64-posix-seh-gcc-12.1.0-mingw-w64msvcrt-10.0.0-r2.zip).
```

Comment:
Brecht Sanders looks like a nice person, but I prefer to have download links from an organization that is larger than 1 single person.

Reply:
For reference, this is the project homepage (which we might want to link to, btw): https://winlibs.com/

Unfortunately the sourceforge project we used thus far doesn't go to gcc 12, so there are little alternatives. the [Mingw-W64 website](https://www.mingw-w64.org/downloads/#mingw-builds) links to [this repo](https://github.com/niXman/mingw-builds-binaries) which is... arguably worse?

Reply:
To be clear, this distribution is used by chocolately and in turn used by GitHub inside their official Windows images. So pretty trustworthy I'd say